contracts/validator-manager/PoSValidatorManager.sol

```diff
@@ -206,7 +206,9 @@ abstract contract PoSValidatorManager is
         bool includeUptimeProof,
         uint32 messageIndex
     ) external {
-        if (!_initializeEndPoSValidation(validationID, includeUptimeProof, messageIndex)) {
+        if (
+            !_initializeEndPoSValidation(validationID, includeUptimeProof, messageIndex, address(0))
```

Comment:
I could overload this function as well, but prefer to keep the API surface a little smaller.